contrib/release.sh

```diff
@@ -1,13 +1,17 @@
 #!/bin/bash -x
```

Comment:
Maybe this can be merged into the make file in the project root.  I'll think about it.